src/main/resources/changelog/scripts/v-1.0.0/resource_graph/tables/resources.sql

```diff
@@ -10,7 +10,12 @@ CREATE TABLE if not exists resources (
   updated_event_id bigint null,
   active_event_id bigint null,
   index_date timestamp,
-  active boolean default true not null
+  active boolean default true not null,
```

Comment:
Moving forward, we have to use ALTER TABLE scripts; Else, deployment to sprint testing env will fail. Earlier, we were in control of the environments, so we could drop the database when such changes are made in DB tables.

Reply:
moved to separate migration.